.github/workflows/deploy-to-azure.yml

```
@@ -0,0 +1,66 @@
+                    - development
+                    - testing
+jobs:
+
+    set-github-environment:
+        runs-on: ubuntu-latest
+        outputs:
+            environment: ${{ steps.set-env.outputs.target_env }}
+        steps:
+            - name: Set Environment
+              id: set-env
+              run: |
```

actionlint (1.7.12)

[error] property "environment_input" is not defined in object type {environment: string}

(expression)